apps/marketplace/src/app/app.component.scss

```diff
@@ -17,3 +17,15 @@
   flex-direction: column;
   height: 100vh;
 }
+
+/*
+ * Make the Component injected by Router Outlet full height:
+ */
+ main {
+  display: flex;
+  flex-direction: column;
+  > *:not(router-outlet) {
+    flex: 1;
+    display: block;
+  }
```

Comment:
I think what you wrote here is 'for every child that is not called router-outlet, add these properties'. But router-outlet is the only child element of main. So you can remove this.

Also, you can add whatever is in `.flex` to `main` and remove the `class="flex"` from the html

Reply:
Corrected.